examples/graphql-caching-global/graphql_caching_global.bal

```diff
@@ -0,0 +1,20 @@
+import ballerina/graphql;
+
+// The `cacheConfig` in `graphql:ServiceConfig` annotation is used to configure the cache for the GraphQL service.
+// The `enabled` field enables/disables the cache for the resource path.
+// The `maxAge` field sets the maximum age of the cache in seconds. (default: 60)
+// The `maxSize` field indicates the maximum capacity of the cache table by entries. (default: 120)
+@graphql:ServiceConfig {
+    cacheConfig: {
+        enabled: true,
+        maxAge: 60,
+        maxSize: 120
```

Comment:
We don't need to use `enabled: true` right? It is the default value. Even the other fields are using the default values, which is redundant. Maybe we can use some other values for those fields here, and maybe mention that you only have to provide values when you need to override the default values.

For example, following should be the default way to enable cache:

```ballerina
@graphql:ServiceConfig {
    cacheConfig: {}
}
```

Maybe we can use another field where we disable the cache for that particular field while having enabled caching at the global-level.

```suggestion
        maxAge: 600,
        maxSize: 100
```